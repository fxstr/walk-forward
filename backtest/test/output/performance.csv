numberOfPositions,24
numberOfWinningPositions,13
numberOfLosingPositions,11
percentProfitable,0.5416666666666666
averageHoldingTimeInDays,16.041666666666668
grossProfit,28324.579569597954
grossLoss,16168.267927000019
profitFactor,1.7518623329031824
averageTradeNetProfit,506.5129851082473
averageWinningTradeProfit,2178.8138130459965
averageLosingTradeLoss,1469.8425388181836
maxWinningTrade,5053.922545300273
maxLosingTrade,5479.328741999998
annualRelativeGrowth,0.1419663558443438
annualRelativeGrowthVariance,0.4662259477320818
maxRelativeDrawdown,0.07480323224250396