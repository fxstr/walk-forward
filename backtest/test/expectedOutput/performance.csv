numberOfPositions,24
numberOfWinningPositions,15
numberOfLosingPositions,9
percentProfitable,0.625
averageBarsHeld,11.041666666666666
grossProfit,32180.026789942764
grossLoss,16097.515692290595
profitFactor,1.9990678937716078
averageTradeNetProfit,670.1046290688404
averageWinningTradeProfit,2145.3351193295175
averageLosingTradeLoss,1788.612854698955
maxWinningTrade,5315.738720931826
maxLosingTrade,4773
annualRelativeGrowth,0.14287345739117674
annualRelativeGrowthVariance,0.47023479037743615
maxRelativeDrawdown,0.07516467339645794